Cargo.toml

```diff
@@ -22,11 +22,11 @@ resolver = "2"
 # `p2pderivatives/rust-dlc#feature/ln-dlc-channels`: 4e104b4. This patch ensures backwards
 # compatibility for 10101 through the `rust-lightning:0.0.116` upgrade. We will be able to drop it
 # once all users have been upgraded and traded once.
-dlc-manager = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "145cf6e" }
-dlc-messages = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "145cf6e" }
-dlc = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "145cf6e" }
-p2pd-oracle-client = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "145cf6e" }
-dlc-trie = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "145cf6e" }
+dlc-manager = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "e3c360" }
+dlc-messages = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "e3c360" }
+dlc = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "e3c360" }
+p2pd-oracle-client = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "e3c360" }
+dlc-trie = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "e3c360" }
```

Comment:
👍 Nice, I think you used the right branch as a base for this!

Reply:
I'm waiting for https://github.com/p2pderivatives/rust-dlc/pull/182 to be merged and will update it again